samples/sample-app.gradle

```diff
@@ -130,11 +130,13 @@ dependencies {
         implementation(project(":datapipelines"))
         implementation(project(":messagingpush"))
         implementation(project(":messaginginapp"))
+        implementation(project(":messaginginapp-compose"))
     } else {
         // Stable releases dependency, use published versions directly
         implementation "io.customer.android:datapipelines:$cioSDKVersion"
         implementation "io.customer.android:messaging-push-fcm:$cioSDKVersion"
         implementation "io.customer.android:messaging-in-app:$cioSDKVersion"
+        implementation "io.customer.android:messaging-in-app-compose:$cioSDKVersion"
```

Comment:
If this is just for demonstration purposes then it's fine. Otherwise, only one of `messaginginapp` or `messaginginapp-compose` should be included on the app side.

Reply:
yup for demonstration, but let me add a comment